content/en/blog/Kairos-at-FOSDEM-2024.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Kairos at FOSDEM 2024"
+date: 2024-02-06T10:53:13+01:00
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+
+Last year [I said](/blog/2023/02/07/kairos-at-fosdem-2023/) you could expect Kairos to engage further during FOSDEM. And I'm quite pleased to say that's exactly what we did! As part of the open-source ecosystem, we recognize the importance of participating in these types of events. We do so because we understand how critical it to go where our users are, but this year we went a bit further and we also reached out to other similar projects.
+
+## From Free-Lunch to Dog-Fooding: A Culinary Journey in Crafting IaC for Kairos Testing and Building
+
+Dimitris did a fantastic job sharing about how we switched our CI/CD pipelines from using GitHub's free workers to our own based on Kairos. You can check out the slides and recordings [here](https://fosdem.org/2024/schedule/event/fosdem-2024-2983-from-free-lunch-to-dog-fooding-a-culinary-journey-in-crafting-iac-for-kairos-testing-and-building/) (as soon as they become available). We were quite pleased with the level of attendance, and we believe that the message came across.
```

Comment:
We can also serve the slides from here as pdf, what do you think? (I mean, upload the file and link to it)

Reply:
@jimmykarily  good idea, can you please give them to me .... also were you able to accept the video? I was doing it and then when hit the finish button it started failing again :(

Reply:
Here:

[From Free-Lunch to Dog-Fooding.pdf](https://github.com/kairos-io/kairos-docs/files/14178246/From.Free-Lunch.to.Dog-Fooding.pdf)

It now says:

<img width="417" alt="image" src="https://github.com/kairos-io/kairos-docs/assets/2794419/76d57fab-f516-4b24-b456-9be54eca6959">

Reply:
slides are on the FOSDEM page, so I won't add here to avoid assets on the website